include/dump_offload.hpp

```diff
@@ -185,8 +185,8 @@ class Handler : public std::enable_shared_from_this<Handler>
                 }
                 return;
             }
-            BMCWEB_LOG_CRITICAL("INFO: Reset OffloadUri of {} dump id {}",
-                                dumpType, entryID);
+            BMCWEB_LOG_INFO("INFO: Reset OffloadUri of {} dump id {}", dumpType,
```

Comment:
Doesn't the BMCWEB_LOG_INFO() already insert "INFO" into the message in the log? Seems like the "INFO:" can be removed from these messages now that they are not output as CRITICAL any more. (I.e. I think it was part of the string before to clarify that it wasn't actually a critical error.)

Reply:
Yeah, that is fair. 
I'll remove the INFO:

Reply:
INFO logs can't be seen in bmc journal, there is no point in logging INFO traces

Reply:
INFO logs can be seen if you increase bmcweb logging level

Reply:
I added this to the agenda for Monday. 
I am going to wait here to remove the "INFO:" until we discuss

Reply:
As discussed in slack, going to go with this